src/wrappers/themis/react-native-themis/android/src/main/java/com/reactnativethemis/ThemisModule.java

```diff
@@ -388,7 +390,6 @@ public void secureMessageSign(String message,
 
     @ReactMethod
     public void secureMessageVerify(ReadableArray message,
-                                    ReadableArray privateKey,
```

Comment:
does it break backward compatibility? do we know that these methods are used anywhere or by someone?

Reply:
A little. Verify method now has only two required parameters. I deleted the middle one - an optional private key. The sign method is compatible.  
I think we should describe the changes in our documentation for react-native-themis. 
The GitHub search tool returns only 8 code results and all results contain in our repositories.

Reply:
> > does it break backward compatibility?
>
> A little.

No such thing. Either applications are going to break after update, or they will not.

Aren't these methods semi-private? As in, they are an implementation detail of ReactNative Themis, and the users will not be calling them directly, but rather through JS code?

Though, I see JS API being changed as well, so it's a genuine breaking change.

---

Please, don't make breaking changes in patch releases, that's irresponsible.

In fact, better avoid making breaking changes at all until the next major version. Deprecations are okay though.

Or else, be honest, slap a prominent red banner in ReactNative documentation that the package is currently under development, is unstable, anything can be changed at any time, and it will be totally the user's fault if their application stops working after an update.

Reply:
Fixed. I returned the middle param privateKey in the public API function. But mark it as unused and optional.